impl/maven-core/src/main/java/org/apache/maven/plugin/PluginParameterExpressionEvaluator.java

```diff
@@ -347,9 +347,7 @@ public File alignToBaseDirectory(File file) {
         // TODO Copied from the DefaultInterpolator. We likely want to resurrect the PathTranslator or at least a
         // similar component for re-usage
         if (file != null) {
-            if (file.isAbsolute()) {
-                // path was already absolute, just normalize file separator and we're done
```

Comment:
How should I preserve this information? Empty business logic as documentation feels overkill.